app/serializers/resource_serializer.rb

```diff
@@ -19,9 +20,13 @@ class ResourceSerializer < ActiveModel::Serializer
 
   belongs_to :default_variant, key: "default-variant", if: -> { object&.resource_type&.name == "metatool" }
 
-  def attributes(*args)
+  def attributes(requested_attrs = nil, reload = false)
```

Comment:
The `reload` parameter doesn't appear to be used here, and I'm not sure where it's coming from. Can you shed some light on that?

Reply:
it's on the overloaded method, if I leave it off I believe it was throwing some errors

Reply:
I take that back, it's not throwing errors, it must have been elsewhere I was seeing the errors